libpromises/processes_select.c

```diff
@@ -548,6 +548,29 @@ static bool SelectProcRegexMatch(const char *name1, const char *name2,
     return false;
 }
 
+static void PrintStringIndexLine(int prefix_spaces, int len)
+{
+    char arrow_str[CF_BUFSIZE];
+    arrow_str[0] = '^';
+    arrow_str[1] = '\0';
+    char index_str[CF_BUFSIZE];
+    index_str[0] = '0';
+    index_str[1] = '\0';
+    for (int lineindex = 10; lineindex <= len; lineindex += 10)
+    {
+        char num[11];
```

Comment:
Right. It's extremely unlikely to be bigger, but `PRINTSIZE` is better practice. Will fix!